deployments/alfajores/.chainId

```diff
@@ -0,0 +1 @@
+44787
```

Comment:
Could /deployments folder be ignored in git?

Reply:
Yes, 👍

Reply:
https://github.com/degen-vc/infinity-contracts/pull/19/commits/10a524c4739811ee21c356d71a7216ad86ce25c1